src/engine/test/source/builder/builders/assetBuilderOutput_test.cpp

```diff
@@ -143,8 +141,8 @@ TEST(AssetBuilderOutput, BuildsOperates)
 {"field":"value"}
 )";
 
-    string filepath1{"/tmp/stageOutputsTest1.txt"};
-    string filepath2{"/tmp/stageOutputsTest2.txt"};
+    string filepath1 {"/tmp/stageOutputsTest1.txt"};
+    string filepath2 {"/tmp/stageOutputsTest2.txt"};
```

Comment:
Isn't it better to have a "define" for this?

Reply:
May be, not on scope on this PR